lib/fs/basicfs_lstat_windows.go

```diff
@@ -67,17 +78,38 @@ func (fi *dirJunctFileInfo) IsDir() bool {
 	return true
 }
 
+type dedupFileInfo struct {
+	os.FileInfo
+}
+
+func (fi *dedupFileInfo) Mode() os.FileMode {
+	// A deduplicated file should be treated as a regular file and not an
+	// irregular file.
+	return fi.FileInfo.Mode() ^ os.ModeIrregular
```

Comment:
```suggestion
	return fi.FileInfo.Mode() &^ os.ModeIrregular
```

should continue working if this bit becomes zero in the future